feconf.py

```diff
@@ -775,6 +786,9 @@ def get_empty_ratings():
 BLOG_DASHBOARD_URL = '/blog-dashboard'
 BLOG_EDITOR_DATA_URL_PREFIX = '/blogeditorhandler/data'
 BULK_EMAIL_WEBHOOK_ENDPOINT = '/bulk_email_webhook_endpoint'
+BLOG_HOMEPAGE_DATA_URL = '/bloghomepagedatahandler/data'
+BLOG_HOMEPAGE_URL = '/blog-homepage'
+BLOG_AUTHORS_PAGE_URL = '/blog-authors'
```

Comment:
What exactly is this page? I don't understand it from the name. Is it a list of blog authors, or blog posts written by a specific author, or something else? If the latter I would have expected it to be still /blog but with a query parameter indicating the parameters of the search.

Reply:
https://xd.adobe.com/view/9bb82409-cdca-432a-b11c-88324643e2c0-ceeb/screen/6ef4fc04-b8e1-4d93-8433-388f92499b6a/

This page. It is completely devoted to the author and contains the name, bio, and the published blog post cards of the blog post published by author.

Reply:
@seanlip changed the authors page url prefix to  '/blog/author'. The complete URL will be 'blog/author/<author_username>'.
PTAL. Thank you.